### ISSUE TITLE
backend for employment profile

### ISSUE THREAD

**Issue by pdpinch** [mitodl/micromasters#240]
Splitting this out from #150 so that @aliceriot and @amir-qayyum-khan can work on it in parallel.

**Comment by pdpinch** [comment]
@aliceriot can you please paste a sample of the json structure you're expecting here?

**Comment by noisecapella** [comment]
Just to be explicit, we need to end up with an `Employment` model with the appropriate fields. The `JSONField` in the PR is just temporary.

**Comment by amir-qayyum-khan** [comment]
@noisecapella @pdpinch so what should I do? Do i need to gather possible fields for Employment from doc or mockup?

**Comment by aliceriot** [comment]
@pdpinch @amir-qayyum-khan  here's what we'd like to see:

```json
profile : {
    work_history: [
        {
            city: "",
            company_name: "",
            country: "",
            end_date: "",
            industry: "",
            position: "",
            start_date: "YYYY-MM-DD"
        }
    ]
```

**Comment by pdpinch** [comment]
Closed via #243

**Comment by amir-qayyum-khan** [comment]
Done this in https://github.com/mitodl/micromasters/pull/243
